pom.xml

```diff
@@ -455,8 +466,8 @@
                                     <shadedPattern>${shadeBase}.com.google.gson</shadedPattern>
                                 </relocation>
                                 <relocation>
-                                    <pattern>com.fasterxml.jackson.core</pattern>
```

Comment:
Can you explain why this needed to change?

Reply:
The current version of avro was 1.7.3, but version 1.11.3 includes dependencies with jackson-core (already relocated) and jackson-databind and jackson-annotations (not relocated).